packages/fx-core/src/component/driver/env/appsettingsGenerate.ts

```diff
@@ -37,6 +38,15 @@ export class GenerateAppsettingsDriver implements StepDriver {
     try {
       this.validateArgs(args);
       const appsettingsPath = getAbsolutePath(args.target, context.projectPath);
+      if (!fs.existsSync(appsettingsPath)) {
+        // try to copy appsettings.json
+        const appsettingsTemplatePath = getAbsolutePath("appsettings.json", context.projectPath);
+        if (!fs.existsSync(appsettingsTemplatePath)) {
+          throw new NoAppsettingsFileUserError(actionName, args.target);
```

Comment:
If appsettings.json not exist, just create an empty json file `{}`

Reply:
Done.